test/CMakeLists.txt

```diff
@@ -173,10 +173,10 @@ else()
 	endif()
 endif()
 
-if(MIOPEN_TEST_GFX908)
-    if(WORKAROUND_ISSUE_1064)
-        list(APPEND SKIP_TESTS test_dropout)
-    endif()
+if(MIOPEN_TEST_GFX908)
+    if(WORKAROUND_ISSUE_1064)
+        list(APPEND SKIP_TESTS test_dropout)
+    endif()
```

Comment:
This is whitespace-only changes or something else useless. Please revert this.

Reply:
Done, thanks.